Dijkstra's Single-Source Shortest Path/Dijkstra.cpp

```diff
@@ -0,0 +1,70 @@
+#include <iostream> 
+#include <limits.h> 
+
+// Define Number of vertices in the graph 
+#define V 7
+
+using namespace std; 
+// function to find the vertex with minimum distance value, from the set of vertices not yet included in shortest path tree 
+int minDistance(int dist[], bool sptSet[]) 
```

Comment:
- I usually prefer working with STL vectors instead of arrays.
- sptSet is a vague variable name and I don't quite understand what it means.